exec.go

```diff
@@ -99,6 +99,10 @@ func execProcess(context *cli.Context) (int, error) {
 	if err != nil {
 		return -1, err
 	}
+	path := context.String("process")
```

Comment:
What if I do `runc exec -p  <container_id> ? 

Wouldn't https://github.com/opencontainers/runc/pull/740/files#diff-cc79fe39318bd23b3e2242c0aaac24fdR149 throw index out of range also?